test/View/Console/DefaultRenderingStrategyTest.php

```diff
@@ -18,10 +18,13 @@
 use Zend\ServiceManager\ServiceManager;
 use Zend\Stdlib\Response;
 use Zend\View\Model;
+use ZendTest\Mvc\EventManagerIntrospectionTrait;
 
 class DefaultRenderingStrategyTest extends TestCase
 {
-    /** @var DefaultRenderingStrategy */
+    use EventManagerIntrospectionTrait;
+
+    /* @var DefaultRenderingStrategy */
```

Comment:
I saw somebody content recently that these hints shouldn't be formal
docblocks. Can you link me to something showing otherwise?
On Oct 1, 2015 6:44 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In test/View/Console/DefaultRenderingStrategyTest.php
> <https://github.com/zendframework/zend-mvc/pull/31#discussion_r40904191>:
>
> >
> >  class DefaultRenderingStrategyTest extends TestCase
> >  {
> > -    /** @var DefaultRenderingStrategy */
> > +    use EventManagerIntrospectionTrait;
> > +
> > +    /* @var DefaultRenderingStrategy */
>
> Should use /**, not /*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-mvc/pull/31/files#r40904191>.
>